polaris-terraform/ui-terraform/nginx.conf

```diff
@@ -12,55 +12,68 @@ http {
 	log_format upstreamlog '[$time_local] $remote_addr - $remote_user - $server_name to: $upstream_addr: $request upstream_response_time $upstream_response_time msec $msec request_time $request_time';
 	access_log  /var/log/nginx/access.log upstreamlog;
 
+  # Note on proxy_pass:
+  # 1) By default proxy_pass will redirect incoming `location /foo` to incoming `location /foo/`
+  # 2) If the string passed to the proxy_pass directive DOES NOT have a slash in it: 
+  #     location /foo/ {
```

Comment:
I think this part of the example should be "location/foo"?

Reply:
I think `location /foo/` is correct, but the wording of the comments and the ordering of the two notes in the comment was poor, and hence misleading.  I have switched it around, hopefully clearer now.